sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnApiDoFnRunner.java

```diff
@@ -2186,6 +2186,7 @@ public TimerMap timerFamily(String timerFamilyId) {
 
     @Override
     public void outputWithTimestamp(OutputT output, Instant timestamp) {
+     // checkTimestamp(timestamp);
```

Comment:
Leftover debugging?

Reply:
This is mostly a WIP for me to figure out what was wrong so not really ready for review. But with this uncommented dataflow on unified worker tests break and with it commented all the other fnapi tests break. So I cannot yet fix it until we fix it for dataflow on unified worker. :)